File: LeetCode/golang/925.go
func isLongPressedName(name string, typed string) bool {
	name_size, typed_size := len(name), len(typed)
	i, j := 0, 0
	for i < name_size && j < typed_size {
		if name[i] == typed[j] {
			i++
			j++
		} else {
			j++
		}
	}
	if i == name_size { 
		return true
	}
	return false
}

File: LeetCode/cpp/925.cpp
class Solution {
public:
    bool isLongPressedName(string name, string typed) {
        int i = 0, j = 0, name_size = name.size(), typed_size = typed.size();
        while (i < name_size && j < typed_size) {
            if (name[i] == typed[j]) {
                ++i;
                ++j;
            } else
                ++j;
        }
        if (i == name_size)
            return true;
        return false;
    }
};